tests/integration/interaction/test_datatable.py

```diff
@@ -43,7 +43,7 @@ def test_editable_changes_data(output_file_url, selenium):
     actions = ActionChains(selenium)
     actions.move_to_element(row_1_cell)
     actions.double_click()
-    actions.send_keys("33\ue007")  # After the backslash is ENTER key
+    actions.send_keys(u"33\ue007")  # After the backslash is ENTER key
```

Comment:
So, the u is necessary for python 2.7 to indicate something as a unicode literal because otherwise string literals in python 2 only contain ascii characters. The \u escape character is recognized by unicode objects. It works without the u in python 3 because all python 3 strings are stored as unicode.